### PR TITLE
feat(ccx): Add CCX project

### PR DESCRIPTION
**Pull request by tumido**
## This Pull Request implements

Adds a CCX project, mapped to `aicoe-ccx` rover group  https://rover.redhat.com/groups/group/aicoe-ccx

### CODE REVIEW

File: manifests/overlays/prod/configs/argo_rbac_cm/policy.csv [hunk 1]
```diff
@@ -61,4 +61,12 @@ p, role:aiops-admin, applications, sync, aiops/*, allow
...
\ No newline at end of file
+g, aicoe-aiops-devops, role:aiops-admin
+
+p, role:ccx-admin, applications, get, ccx/*, allow
+p, role:ccx-admin, applications, update, ccx/*, allow
+p, role:ccx-admin, applications, delete, ccx/*, allow
+p, role:ccx-admin, applications, sync, ccx/*, allow
+p, role:ccx-admin, applications, override, ccx/*, allow
+p, role:ccx-admin, applications, action/*, ccx/*, allow
+g, ccx-dev, role:ccx-admin
```
Comment (HumairAK) [inline manifests/overlays/prod/configs/argo_rbac_cm/policy.csv:72]: the configs are a special situation where we don't add the newline, do you mind removing it again

Comment (HumairAK) [reply manifests/overlays/prod/configs/argo_rbac_cm/policy.csv:72]: actually I think it's fine for policy.csv

Comment (tumido) [reply manifests/overlays/prod/configs/argo_rbac_cm/policy.csv]: Good to know, I've fixed it anyways. :+1:

Comment (anishasthana) [inline manifests/overlays/prod/configs/argo_rbac_cm/policy.csv:72]: I think argocd admins should have "get" access to this project as well.

Comment (tumido) [reply manifests/overlays/prod/configs/argo_rbac_cm/policy.csv:72]: Good call. I think aiops should have access to it as well. :slightly_smiling_face:

Comment (tumido) [reply manifests/overlays/prod/configs/argo_rbac_cm/policy.csv:72]: Actually, argocd admins shoul have access to all applications, no matter what project. We should add a generic rule for that.

Comment (tumido) [reply manifests/overlays/prod/configs/argo_rbac_cm/policy.csv:72]: Additionally argocd-admins already have access to all projects:

https://github.com/AICoE/aicoe-cd/blob/704dc0366b63ac873ff9303ad47d1f0ca7e2654a/manifests/overlays/prod/configs/argo_rbac_cm/policy.csv#L10-L12

**Comment by anishasthana** [comment]
Have we made it clear that the argocd usage is for Argo workflows?

**Comment by tumido** [comment]
Sure! :)so far we aim for 1 or 2 projects originated from CCX to be running Argo Workflows - this is just for monitoring purposes, and isolated in a separate project. If we want to expand this colab in future, we'll reiterate.

**Comment by tumido** [comment]
This PR is ready to reviews. :slightly_smiling_face:

**Review by anishasthana — APPROVED** [review]
lgtm

**Comment by tumido** [comment]
cough cough, any idea why's sesheta not merging?

**Pull request merged**